code/game/machinery/buttons.dm

```diff
@@ -56,6 +56,12 @@
 		else
 			icon_state = skin
 
+/obj/machinery/button/allowed(mob/M)
+	if(req_access_txt == "syndicate" && issilicon(M)) //The syndicate figured out how to lock out borgs

```

Comment:
req_access_txt format is "1;23;45" so this is still incorrect. You should be checking the current access not the map helper one too.

Reply:
@Robustin pls get this tho

Reply:
I'm not sure I understand what @AnturK means by "map helper one". Do I need to look up the numerical definition for "syndicate"?

Reply:
_txt accesses are only used on initialization to pass mapped in values, the current values are stored in the ones without _txt